Wed Jul 03 01:01:45, Warn:  Output directory does not exist yet. We therefore create it now. 
Traceback (most recent call last):
  File "/dss/dsshome1/01/di35poq/.local/bin/seissol_output_extractor", line 8, in <module>
    sys.exit(main())
  File "/dss/dsshome1/01/di35poq/.local/lib/python3.8/site-packages/seissolxdmfwriter/seissol_output_extractor.py", line 199, in main
    sxw.write_from_seissol_output(
  File "/dss/dsshome1/01/di35poq/.local/lib/python3.8/site-packages/seissolxdmfwriter/seissolxdmfwriter.py", line 482, in write_from_seissol_output
    ) = sx.GetDataLocationPrecisionMemDimension(name)
  File "/dss/dsshome1/01/di35poq/.local/lib/python3.8/site-packages/seissolxdmfwriter/seissol_output_extractor.py", line 139, in GetDataLocationPrecisionMemDimension
    return super().GetDataLocationPrecisionMemDimension(dataName)
  File "/dss/dsshome1/01/di35poq/.local/lib/python3.8/site-packages/seissolxdmf/seissolxdmf.py", line 131, in GetDataLocationPrecisionMemDimension
    raise NameError("%s not found in dataset" % (dataName))
NameError: Mud not found in dataset
